src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -136,7 +136,8 @@ public async Task PostExport()
 
                 var contentDisposition = "attachment; filename=" + Uri.EscapeDataString(fileName) + ".ravendbdump";
                 HttpContext.Response.Headers["Content-Disposition"] = contentDisposition;
-
+                HttpContext.Response.Headers["Content-Type"] = "text/ravendbdump";
```

Comment:
did we already use this somewhere or you create this content type? I wonder if it is more safe to use some generic content type?

Reply:
done - changed to `text/plain`